dqc_us_rules/dqc_us_0018.py

```diff
@@ -73,7 +73,7 @@ def _load_cache(val):
                 val.usgaapDeprecations = json.load(file)
                 file.close()
 
-            except FileNotFoundError:
+            except FileNotFoundError:  # noqa
```

Comment:
This weirds me out about flake8, to be honest, I wonder why it isn't seeing the builtin?